src/clusterfuzz/_internal/build_management/build_manager.py

```diff
@@ -1116,6 +1188,9 @@ def setup_regular_build(revision,
 
     return None
 
+  # build_url points to a GCP bucket, and we're only converting it to its HTTP
+  # endpoint so that we can use remote unzipping.
+  http_build_url = build_url.replace('gs://', 'https://storage.googleapis.com/')
```

Comment:
Optional: now that the transformation is so simple, you could push it into `HttpZipFile` if you wanted.

Reply:
That's true, although I think we should rather push it in the `_unpack` function maybe? Let's keep it here for now, there will be subsequent changes once we validate that this works as expected.